fusionSuite/backend
# Add auditlogs feature

Changes proposed in this pull request:

- put in audits table all modification of items, connections and all failed events

How to test the feature manually:

1. login and check in table `audits`
2. create an item and check in table `audits`
3.update an item and check in table `audits`

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._

Related to #
